packages/mds-cache/index.ts

```diff
@@ -152,9 +152,9 @@ async function hgetall(key: string): Promise<CachedItem | CachedHashItem | null>
   return null
 }
 
-async function getVehicleType(keyID: UUID): Promise<VEHICLE_TYPE> {
-  // TODO: Fix type entry into cache so we don't need to do this unkown conversion
-  return ((await getClient()).hgetAsync(decorateKey(`device:${keyID}:device`), 'type') as unknown) as VEHICLE_TYPE
+async function getVehicleType(keyID: UUID): Promise<VEHICLE_TYPE | null> {
```

Comment:
I don't prefer the double `await`. I think `const client = await getClient()` would be more readable.